components/team/team.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames/bind';
+import styles from './team.module.scss';
+import { TeamItemI } from './team-item.interface';
+import Image from 'next/image';
+import { H3, Text } from '@components/typography/typography';
+
+const cx = classNames.bind(styles);
+
+type Props = {
+  items: TeamItemI[];
+  basePath: string;
+  style?: string;
+};
+
+const Team = ({ style, basePath, items }: Props) => {
+  const teamItems = items.map((item: TeamItemI) => (
+    <div key={item.name}>
+      <Image
+        className={styles.images}
```

Comment:
Since you're already binding it, you can probably just use `cx('images')`.

In addition, by the looks of it, the styling of `border: none` appears to come from [next/image](https://github.com/vercel/next.js/discussions/18312#discussioncomment-407751). To get around it, you should consider applying the border and radius outside the image by doing something like this:

```
<div className={cx('images')}>
	<Image ... />
</div>
```

(You will also need to add an overflow:hidden to ensure the image does not bleed out of the rounded corners)

Reply:
👍